stuffioAPI/stuffioAPI/Migrations/20180111112406_SeedDatabase.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace stuffioAPI.Migrations
+{
+    public partial class SeedDatabase : Migration

```

Comment:
WUT?

Reply:
Sorry bout that haha, made a context seeder . removed this migration